module-services/service-cellular/call/tests/test-CallMachine.cpp

```diff
@@ -362,6 +364,7 @@ TEST_CASE("call outgoing - answered")
     REQUIRE(machine->machine.process_event(call::event::StartCall{CallType::CT_OUTGOING, number.getView()}));
     REQUIRE(machine->machine.process_event(call::event::Answer{}));
     fakeit::Verify(Method(di.audio, play)).Exactly(0);
+    fakeit::Verify(Method(di.gui, notifyOutgoingCallAnswered)).Exactly(1);
```

Comment:
it's already propagating:
```
            di.gui->notifyCallStarted(call.record.phoneNumber, call.record.type);
```
why do we have to have another state here? if we do - can't we remove its notification in HandleCallStart? So that it would be handled in a similar manner in UI and in Bluetooth?

Also: for some reason, we have multicast in GUI and in multicast.
By design - GUI was for actions ( first two calls in GUI dependency) then we have multicast. This might introduce redundancies - can you check it?

Reply:
call started is called for both outgoing and incoming calls - that was the case why we needed the refactor TBH :)

For the placement - I just went with the scheme that was already in the code